sdk/src/beta9/abstractions/base/__init__.py

```diff
@@ -69,9 +70,10 @@ def __init_subclass__(cls, /, **kwargs):
         frames = inspect.stack()
         frame = frames[-1]
 
-        if frame.code_context and any(
-            substr in frame.code_context[0] for substr in ("import beam", "from beam")
-        ):
+        if (
+            frame.code_context
+            and any(substr in frame.code_context[0] for substr in ("import beam", "from beam"))
+        ) or "beam" in sys.modules:
```

Comment:
@nickpetrovic @jsun-m I'm curious what you guys think about removing the frame logic and just checking the sys modules.

Reply:
I think that's a good idea. If its there every time when beam is calling beta9 and its not there when only beta9 is being invoked via the CLI, then I say we do it.

Reply:
I am going to test this a bit more, but I am fairly sure its safe